src/styles/blocks/_stopwatch.scss

```diff
@@ -0,0 +1,38 @@
+.stopwatch {
+  @include circle($stopwatch-size);
+
+  position: relative;
+  display: flex;
+  justify-content: center;
+
+  background: radial-gradient(#020024, #d72644 27%, #05c3ff 47%, #010024 69%);
+  border: 8px dotted #fff;
+
+  &__center {
+    @include circle($center-size);
+
+    position: absolute;
+    bottom: calc(50% - #{$center-size} / 2);
+    z-index: 3;
+    background-color: #010024;
+    border: 1px solid #fff;
+  }
+
+  &__second-hand {
+    @include hand($second-hand-heigth, $second-hand-width);
+
+    bottom: $vertical-align;
+    z-index: 2;
+    background-color: $hand-color;
+    animation: 5s linear infinite hand-move;
```

Comment:
Consider moving the animation duration (5s) to a variable for better maintainability.